.github/scripts/config.sh

```diff
@@ -4,13 +4,22 @@ set -e # exit on error
 set -x # echo on
 set -o pipefail # fail of any command in pipeline is an error
 
+# Branches that will be used for build when UPDATE_SOURCES=1.
 BINUTILS_BRANCH=${BINUTILS_BRANCH:-woarm64}
 GCC_BRANCH=${GCC_BRANCH:-woarm64}
 MINGW_BRANCH=${MINGW_BRANCH:-woarm64}
 CYGWIN_BRANCH=${CYGWIN_BRANCH:-main}
 CYGWIN_PACKAGES_BRANCH=${CYGWIN_PACKAGES_BRANCH:-main}
 COCOM_BRANCH=${COCOM_BRANCH:-master}
 
+# Baseline branches used for rebase when REBASE_SOURCES=1.
+BINUTILS_BASE_BRANCH=woarm64
+GCC_BASE_BRANCH=woarm64
+MINGW_BASE_BRANCH=woarm64
```

Comment:
why it should be different than UPDATE_SOURCES?

Reply:
\`\*\_BRANCH\` are personal feature branches one is working on. \`\*\_BASE\_BRANCH\` are shared development branches that are being updated by the whole team meanwhile. \`UPDATE\_SOURCES=1\` just updates the local copies of the feature branches to their remote versions. \`UPDATE\_SOURCES=1\` together with \`REBASE\_SOURCES=1\` also rebases the features branches to the development branches.